extensions/everhour/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "everhour",
+  "title": "Everhour",
```

Maybe add a little bit more context here? The extension title is used in the store.
```suggestion
  "title": "Everhour Time Tracking",
```

---

Good shout!